### PR TITLE
Create common.ps1 to simplify Windows integration tests

### LINKED ISSUE #11949: Create Windows common.ps1
We need to make all the Windows tests more generic - we can do this the same way common.sh works.

### PR DESCRIPTION
**Pull request by andriyDev**
fixes #11949.

### CODE REVIEW

File: hack/jenkins/windows_integration_teardown.ps1 [hunk 1]
```diff
@@ -14,4 +14,12 @@
 
 $test_home="$env:HOMEDRIVE$env:HOMEPATH\minikube-integration"
 
+if ($driver == "docker") {
```
Comment (sharifelgamal) [inline hack/jenkins/windows_integration_teardown.ps1:17]: the `==` should be an `-eq`

huge shoutout to powershell

Comment (andriyDev) [reply hack/jenkins/windows_integration_teardown.ps1]: >:(

Done haha, good spot!

**Review by sharifelgamal — APPROVED** [review]

**Pull request merged**